osu.Game/Modes/Objects/Types/IHasDistance.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+namespace osu.Game.Modes.Objects.Types
+{
+    /// <summary>
+    /// A HitObject that has a distance.

```

Comment:
`A HitObject that has a positional length.`

Try and describe interfaces using synonyms to add more clarity (similar to how a thesaurus does). Using "distance" alone may be ambiguous to some.